config/routes.rb

```diff
@@ -14,6 +14,8 @@
     resources :sessions, only: [:new, :create, :destroy]
     resource :password_reset, path: '/password-reset', except: [:destroy, :edit]
 
+    resource :profile
```

Comment:
Since you can't really create or delete a profile, I'd probably suggest we restrict these routes to `:edit` and `:update`.